### PR TITLE
[4.3][Guided Tours] Cleanup Tours model

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
This PR makes several clean up to Tours module from Guided Tours extension 

- No need to override `getEmptyStateQuery` method from parent class (This override contains really wrong code, seems copied from Steps model)
- Remove getTable method. We do not need this method from a List model

### Testing Instructions
- Install Joomla from download package generated by this PR (to make sure Guided Tours is available on your installation). Access to Guided Tours, see list of tours and make sure nothing is broken from this page 

### Actual result BEFORE applying this Pull Request
Unnecessary wrong code in Tours model

### Expected result AFTER applying this Pull Request
The unnecessary wrong code removed.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by joomdonation** [comment]
Question for @obuisard : In this model, we also have code to handle extensions filter (no idea how it should work). However, that filter is not in filter form, so should we remove that code as well ?

**Comment by obuisard** [comment]
> Question for @obuisard : In this model, we also have code to handle extensions filter (no idea how it should work). However, that filter is not in filter form, so should we remove that code as well ?

No, it is used by the Guided Tours module to show tours according to a specific extension that is showing on the screen.

**Comment by joomdonation** [comment]
> No, it is used by the Guided Tours module to show tours according to a specific extension that is showing on the screen

Ah, OK. Thanks for the info.

File: administrator/components/com_guidedtours/src/Model/ToursModel.php [hunk 1]
```diff
@@ -58,38 +58,6 @@ public function __construct($config = [])
...
-     *
-     * @return DatabaseQuery
-     *
-     * @since __DEPLOY_VERSION__
-     */
-    protected function getEmptyStateQuery()
-    {
-        $query = clone $this->_getListQuery();
-
-        if ($query instanceof DatabaseQuery) {
```
Comment (Quy) [inline administrator/components/com_guidedtours/src/Model/ToursModel.php:72]: Please remove import statement.

**Comment by obuisard** [comment]
I have tested this item :white_check_mark: successfully on 17b1444703e94137d4bd07aa507e967646a4c3af<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40003">issues.joomla.org/tracker/joomla-cms/40003</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40003">issues.joomla.org/tracker/joomla-cms/40003</a>.</sub>

**Comment by obuisard** [comment]
Thank you Tuan @joomdonation!

**Pull request merged**